Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -602,6 +602,86 @@ public void When_both_field_and_properties_are_configured_for_inclusion_both_sho
             // Assert
             act.Should().Throw<XunitException>().Which.Message.Should().Contain("Field1").And.Contain("Property1");
         }
+
+        [Fact]
+        public void Taking_internal_fields_into_consideration()
+        {
+            // Arrage
+            var subject = new ClassWithAllAccessModifiersForMembers("Lorem", "protectedValue", "ipsum", "protectedInternalValue", "privateValue", "privateProtectedValue");
```

Comment:
🔧 I find this example too hard to read. I'd prefer that you introduce a dedicated test class directly under the tests.